INSTALL.md

```diff
@@ -4,13 +4,13 @@ The easiest and recommended way of installing the CLI on all platforms (Windows,
 to use the [Standalone Installation](#standalone-installation) described below. Some specific CLI
 commands have a few extra installation steps: see section [Additional Dependencies](#additional-dependencies).
 
-> **Windows users:** we now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
+> **Windows users:** We now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
```

Comment:
Almost all changes in this PR are to address @hedss' late review of #1175. See my other comment for a new change that I am proposing.